mapper/tedge_mapper/src/sm_c8y_mapper/mapper.rs

```diff
@@ -405,6 +453,34 @@ mod tests {
         assert_eq!(res, "https://test_host/inventory/managedObjects/12345");
     }
 
+    #[rstest]
+    #[case("http://aaa.test.com")]
+    #[case("https://aaa.test.com")]
+    #[case("ftp://aaa.test.com")]
+    #[case("mqtt://aaa.test.com")]
+    #[case("https://t1124124.test.com")]
+    #[case("https://t1124124.test.com:12345")]
+    #[case("https://t1124124.test.com/path")]
+    #[case("https://t1124124.test.com/path/to/file.test")]
+    #[case("https://t1124124.test.com/path/to/file")]
+    fn url_is_my_tenant_correct_urls(#[case] url: &str) {
+        let tenant_uri = "test.abc.com";
```

Comment:
Something is wrong here. To have these tests passing, I would expect matching domain names!

Reply:
Yes, it looks like GH actions didn't run on this PR for some reason, and in terms of code it is an oversight with commit now updated.